### DISCUSSION TITLE
关于E站配额

### DISCUSSION THREAD

**Discussion by HNPA2018** [xiaojieonly/Ehviewer_CN_SXJ#1450] [Q&A]
大概一个月前的一天剩余的配额突然就变成0/0了，下载也下不了，阅读就显示配额不足，更换梯子节点和里站账号也一样，刷新配额就会显示parse image limits error，想请教一下大佬们可能是什么问题，需要如何解决。
![Screenshot_2024-10-13-00-52-25-808_com.xjs.ehviewer.jpg](https://github.com/user-attachments/assets/9533ff77-2527-49d1-96f5-aaa538c63647)

**Comment by Pony-CW** [comment]
https://github.com/FooIbar/EhViewer/issues/1563#issuecomment-2310031101
